MEMBERSHIP.md

```diff
@@ -65,20 +67,26 @@ Reviewers are knowledgeable about the codebase and are able review code for qual
 
 To become a project Reviewer, you must meet the following requirements:
 
+* Member for at least **3 months**.
+* Primary reviewer or author for at least **5 substantial PRs** to the codebase.
```

Comment:
I'm not sure we want to put numbers in here yet. It might make sense to decrease each of the numbers to one and add a note at the bottom of this section that as OSCAL compass matures we expect the PR review / author count to increase.